package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "start": "concurrently -k -r 'rollup -c -w' 'wait-on script.js style.css && zcli themes:preview'",
     "build": "rollup -c",
+    "prebuild": "rm assets/*-bundle.js",
```

Comment:
Note: we need to use the `-f` option to avoid errors when there are no files matching the pattern

Reply:
Addressed in [8144437](https://github.com/zendesk/copenhagen_theme/pull/465/commits/8144437fada1f2244e7d99eae7d2c95778acaf79). Should also fix build.